include/SANDTrackerCell.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "SANDWireInfo.h"
+
+class SANDTrackerPlane;
+
+class SANDTrackerCellID : public SingleElStruct<unsigned long>
+{
+ public:
+  SANDTrackerCellID(unsigned long id) : SingleElStruct<unsigned long>(id){};
+  SANDTrackerCellID() : SingleElStruct<unsigned long>(){};
+};
+
+class SANDTrackerCell
+{
+  SANDTrackerCellID _id;
```

Comment:
several standards are used in this merge requests for class data members. Please consider standardizing the naming convention

https://google.github.io/styleguide/cppguide.html#Variable_Names